### PR TITLE
Added New Blog 

### LINKED ISSUE #1196: [NG] Add latest blog to layer5-ng
**Current Behavior**
The [Layer5 at Kubecon blog](https://layer5.io/blog/layer5-at-kubecon-na-2020) has been added recently to layer5 site. 

**Desired Situation**
Migrate the blog to layer5-ng site as well. The blogs on layer5-ng can be found [here](https://layer5ng.netlify.app/blog).

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by akashgp09**
**Description**

This PR fixes #1196

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/blog/2020-11-17-layer5-at-kubecon-2020/index.mdx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+<h4 style="color:#00b39f;">
+  KubeCon + CloudNativeCon NA 2020 is here and Layer5 is in the spotlight!
+</h4>
+
+<a href="https://kccncna20.sched.com/event/ekEz/service-mesh-specifications-and-why-they-matter-in-your-deployment-lee-calcote-naveen-kumar-layer5">
+  <b>Service Mesh Specifications and Why They Matter in your Deployment</b>{" "}
+  <img class="image-right" style="width:28%;" src={kubecon1} />
+</a>
+
+<h6 style="color:gray;"> November 18, 2020 </h6>
```
Comment (Jashpatel1) [inline src/collections/blog/2020-11-17-layer5-at-kubecon-2020/index.mdx:34]: This title sizes are smaller than normal text. Can you please look into it?

File: src/collections/blog/2020-11-17-layer5-at-kubecon-2020/index.mdx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+the challenge of interoperability, workload and performance management between service
+meshes: Service Mesh Performance (SMP) and Service Mesh Interface (SMI), Multi-Vendor
+Service Mesh Interoperation (Hamlet). Attend to learn what makes each of these specifications
+unique and why they are very much needed.
+
+<a href="https://kccncna20.sched.com/event/etsE/cncf-sig-network-intro-deep-dive-lee-calcote-layer5">
+  <b>CNCF SIG Network: Intro and Deep-Dive</b>
+</a>
+
+<h6 style="color:gray;"> November 20, 2020 </h6>
```
Comment (Jashpatel1) [inline src/collections/blog/2020-11-17-layer5-at-kubecon-2020/index.mdx:46]: Same as above

File: src/collections/blog/2020-11-17-layer5-at-kubecon-2020/index.mdx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+<a href="https://kccncna20.sched.com/event/fMYB/meet-the-maintainer-service-mesh-interface">
+  <img class="image-right" style="padding-top:8px;width:28%;" src={kubecon2} />
+</a>
+
+<br />
+<a href="https://kccncna20.sched.com/event/fMYB/meet-the-maintainer-service-mesh-interface">
+  <b>Meet the Maintainers: Service Mesh Interface</b>
+</a>
+
+<h6 style="color:gray;"> November 17, 2020 </h6>
```
Comment (Jashpatel1) [inline src/collections/blog/2020-11-17-layer5-at-kubecon-2020/index.mdx:69]: Same as above

**Comment by Jashpatel1** [comment]
Also, can you explain about the huge deletion in package-lock.json? I am concerned if it breaks somewhere.

File: src/collections/blog/2020-11-17-layer5-at-kubecon-2020/index.mdx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+<h4 style="color:gray;"> November 17, 2020 </h4>
+Join the Service Mesh Interface (SMI) and Meshery maintainers for a Meet the Maintainers
+office hours. Learn all about Service Mesh Interface and how Meshery is the service
+mesh manager to determine the conformance of all service meshes in or out of compliance
+with the specification.
+<br />
+<br />
+<br />
+
+<h6 style="color:#3c494f;text-align:center;">
```
Comment (Jashpatel1) [inline src/collections/blog/2020-11-17-layer5-at-kubecon-2020/index.mdx:82]: The size of this text is also to be improved.

Comment (Jashpatel1) [reply src/collections/blog/2020-11-17-layer5-at-kubecon-2020/index.mdx:82]: > The size of this text also needs to increase.

**Comment by Jashpatel1** [comment]
> Also, can you explain about the huge deletion in package-lock.json? I am concerned if it breaks somewhere.

**Comment by akashgp09** [comment]
@Jashpatel1

File: src/collections/blog/2020-11-17-layer5-at-kubecon-2020/index.mdx [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+---
+
+import { BlogWrapper } from "../Blog.style.js";
+import kubecon1 from "./kubecon1.jpg";
+import kubecon2 from "./kubecon2.png";
+import kubecon3 from "./kubecon3.png";
+
+<BlogWrapper>
+
+<p style="color:#3c494f;text-align:center;">
```
Comment (Tanuj22) [inline src/collections/blog/2020-11-17-layer5-at-kubecon-2020/index.mdx:19]: Do not use inline styles (do this for other places also)

**Comment by akashgp09** [comment]
@Jashpatel1 @Tanuj22 how to resolve the conflicting file **package-lock.json** ?

**Comment by Nikhil-Ladha** [comment]
Remove the file from the commit.

**Review by Jashpatel1 — APPROVED** [review]
Thanks, @akashgp09 . Great work here.

**Pull request merged**